LSP-some-sass.sublime-settings

```diff
@@ -0,0 +1,146 @@
+{
+	"command": ["${node_bin}", "${server_path}", "--stdio"],
```

Comment:
I usually place it at the bottom since it's not really encouraged for user to change this one.

Reply:
Done!